.github/workflows/run_tests.yaml

```diff
@@ -93,25 +93,26 @@ jobs:
 
       # Step 6: Full PyTest
       - name: Test with pytest
-        env:
-          COVERALLS_FLAG_NAME: ${{ matrix.test-name }}
-          COVERALLS_PARALLEL: true
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
         run: |
-          pip install coveralls
-          pytest . -v --cov AxonDeepSeg/ --cov-report term-missing
-          coveralls --service=github
+          pytest ./test/ -v --cov AxonDeepSeg/ --cov-report=lcov
```

Comment:
Just to fully understand this change:

- `lcov` has been added: Coveralls GitHub Action [requires LCOV format](https://github.com/coverallsapp/github-action?tab=readme-ov-file#usage).
- `term-missing` has been removed: Why is this?

Reply:
>term-missing has been removed: Why is this?

Ahh, that's just to show which lines are being missed in the CLI run log on github actions. It's not really a requirement either way, it doesn't impact the coverage report or the tests themselves, just a style choice. I can reintegrate it if you think other ppl might want it when looking at the GitHub actions runs